src/platform/landing-pages/dev-template.ejs

```diff
@@ -35,14 +35,25 @@
     <%= modifyScriptTags(htmlWebpackPlugin.tags.bodyTags) %>
 
     <script nonce="**CSP_NONCE**" type="text/javascript">
+      function scrollToContent(e) {
+        e.preventDefault();
+        e.target.hidden = true;
+        const contentRoot = document.querySelector('#content');
+        const contentH1 = document.querySelector('#content h1');
+        if (contentH1) {
+          window.scrollTo(0, contentH1.offsetTop);
```

Comment:
We'll need to also set focus to the element. I assume we're fixing the IE11 skip link bug with this?

Process should be:
1) click the link
2) set `tabindex="-1"` on the target element ( `contentH1.setAttribute('tabindex', '-1')` )
3) Programmatically move focus to the target element ( `contentH1.focus()` )
4) Set blur event on the target element to remove the `tabindex="-1"` to clean up the DOM and make sure the element can't take focus again

Reply:
`main` is usually the common practice, but we recommended the `h1` to address `tab` hiccups on VO/Safari and to prevent folks having to wade through the breadcrumbs. [Deque provides an example of setting focus to the `h1` here](https://dequeuniversity.com/tips/add-skip-navigation-link). @AngelaFowler82 can provide more context as a screen reader user! 😄

Reply:
Also can confirm the need to set focus following Tim's guidance! Not sure if there's a javascript function in the platform code somewhere though.

Reply:
I have added code to set focus.

Reply:
I'm assuming the answer is "yes" based on your comment "can confirm the need to set focus following Tim's guidance".